yourgov/components/FormMobileFoodVendorPermit/FormTask1Step1.tsx

```diff
@@ -0,0 +1,285 @@
+import { Details } from '@ag.ds-next/react/details';
+import { useScrollToField } from '@ag.ds-next/react/field';
+import { PageAlert } from '@ag.ds-next/react/page-alert';
+import {
+	SummaryList,
+	SummaryListItem,
+	SummaryListItemDescription,
+	SummaryListItemTerm,
+} from '@ag.ds-next/react/summary-list';
+import { FormRequiredFieldsMessage } from '../FormRequiredFieldsMessage';
+import { useGlobalForm } from './GlobalFormProvider';
+import { FormActions } from './FormActions';
+import { FormTask1Container } from './FormTask1Container';
+import { useFormTask1Context } from './FormTask1Provider';
+import {
+	task1Step1FormSchema,
+	Task1Step1FormSchema,
+	task1Step1Part2FormSchema,
+	Task1Step1Part2FormSchema,
+} from './FormTask1FormState';
+
+export function FormTask1Step1() {
+	const { isSideFlow, setIsSideFlow, formState } = useGlobalForm();
+
+	const scrollToField = useScrollToField();
+	const errorRef = useRef<HTMLDivElement>(null);
+	const headingRef = useRef<HTMLHeadingElement>(null);
+	const [focusedError, setFocusedError] = useState(false);
+
+	const defaultValues = formState.task1?.step1;
+
+	const {
+		register,
+		handleSubmit,
+		reset,
+		formState: { errors },
+	} = useForm<Task1Step1FormSchema>({
+		defaultValues: defaultValues,
+		resolver: yupResolver(task1Step1FormSchema),
+	});
+
+	const [hasClosedForm, setHasClosedForm] = useState(false);
+	const [localFormState, setLocalFormState] = useState(defaultValues);
+	const [isSaving, setIsSaving] = useState(false);
+
+	const onSave: SubmitHandler<Task1Step1FormSchema> = (data) => {
+		setFocusedError(false);
+		setIsSaving(true);
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
+		setLocalFormState(data);
+		// Using a `setTimeout` to replicate a call to a back-end API
+		setTimeout(() => {
+			setIsSaving(false);
+			setIsSideFlow(false);
+			setHasClosedForm(true);
+		}, 1500);
+	};
+
+	function onDiscardChangesClick() {
+		reset();
+		setIsSideFlow(false);
+		setHasClosedForm(true);
+	}
+
+	function changeDetails() {
+		setIsSideFlow(true);
+	}
+
+	useEffect(() => {
+		if (!hasClosedForm) return;
+		headingRef.current?.focus();
+		setHasClosedForm(false);
+	}, [hasClosedForm]);
+
+	const onError: SubmitErrorHandler<Task1Step1FormSchema> = () => {
+		setFocusedError(false);
+	};
+
+	// Only show the page alert if there is more than 1 error
+	const hasErrors = Object.keys(errors).length > 1;
+
+	useEffect(() => {
+		if (hasErrors && !focusedError) {
+			errorRef.current?.focus();
+			setFocusedError(true);
+		}
+	}, [hasErrors, focusedError, errors]);
+
+	return (
+		<FormTask1Container
+			formTitle="Owner details"
+			formIntroduction="Confirm your name and contact details."
+			formCallToAction={isSideFlow ? <FormRequiredFieldsMessage /> : undefined}
+		>
+			<Stack gap={3} alignItems="flex-start" width="100%">
+				{isSideFlow ? (
+					<Stack gap={1.5} width="100%">
+						<H2>Update personal details</H2>
```

Comment:
Why does this step lose the progress nav? And the back direction link is weird taking you back to the Task List page.